tests/sentry/search/events/builder/test_metrics.py

```diff
@@ -1452,84 +1451,6 @@ def test_multiple_dataset_but_no_data(self):
         # Handled by the discover transform later so its fine that this is nan
         assert math.isnan(data["p50"])
 
-    @mock.patch("sentry.search.events.builder.discover.raw_snql_query")
-    @mock.patch("sentry.search.events.builder.metrics.indexer.resolve", return_value=-1)
-    def test_dry_run_does_not_hit_indexer_or_clickhouse(self, mock_indexer, mock_query):
-        query = MetricsQueryBuilder(
-            self.params,
-            # Include a tag:value search as well since that resolves differently
-            query=f"project:{self.project.slug} transaction:foo_transaction",
-            dataset=Dataset.PerformanceMetrics,
-            selected_columns=[
-                "transaction",
-                "p95(transaction.duration)",
-                "p100(measurements.lcp)",
-                "apdex()",
-                "count_web_vitals(measurements.lcp, good)",
-            ],
-            dry_run=True,
-        )
-        query.run_query("test_query")
-        assert not mock_indexer.called
-        assert not mock_query.called
-
-    @mock.patch("sentry.search.events.builder.metrics.indexer.resolve", return_value=-1)
-    def test_multiple_references_only_resolve_index_once(self, mock_indexer):
```

Comment:
Is this still a valid test? It wasn't setting `dry_run=True` in the constructor

Reply:
Ah you're right :+1: